blessedDeps.gradle

```diff
@@ -13,6 +13,9 @@ rootProject.ext.JAVA_TARGET_VERSION = JavaVersion.VERSION_1_8
 rootProject.ext.TARGET_SDK_VERSION = 28
 rootProject.ext.COMPILE_SDK_VERSION = 28
 rootProject.ext.MIN_SDK_VERSION = 14
+rootProject.ext.COMPOSE_TARGET_SDK_VERSION = 30
```

Comment:
ideally we bump the target/compile versions for all the other modules too and only differ on min, but we can clean that up later